UPGRADE.md

```diff
@@ -30,6 +30,11 @@ As a consequence, automatic cache setup in Doctrine\ORM\Tools\Setup::create*Conf
 - Memcached extension (ext-memcached) will be used instead of obsolete Memcache (ext-memcache).
 - XCache support was dropped as it doesn't work with PHP 7.
 
+## Deprecated: `Doctrine\ORM\EntityManagerInterface#copy()`
```

Comment:
This section should be above `# Upgrade to 2.6` and have a title `# Upgrade to 2.7`

Reply:
The new version headline is created and the deprecation text is assigned to it.